kuma/core/whitenoise/KumaDjangoWhiteNoise.py

```diff
@@ -0,0 +1,15 @@
+from whitenoise.django import DjangoWhiteNoise
+
+
+class KumaDjangoWhiteNoise(DjangoWhiteNoise):
+
+    # Custom MIME types
+    KUMA_MIMETYPES = (
+        ('application/font-woff', '.woff'),
+        ('application/font-woff2', '.woff2'),)
+
+    # Use our custom MIME types, together with the default MIME types, in
+    # WhiteNoise. When our MIME types conflict with the defaults, use ours.
+    EXTRA_MIMETYPES = tuple({e[1]: e for e in reversed(
+        KUMA_MIMETYPES + DjangoWhiteNoise.EXTRA_MIMETYPES
+    )}.values())
```

Comment:
Ough, yeah, I'd rather just redefine the tuple completely in this case. This magic is not worth the effort given the fact that the upstream whitenoise class currently only has one item: https://github.com/evansd/whitenoise/blob/bdeea1a18b134ba59a9f6a31dd2f1839836b4b14/whitenoise/base.py#L72-L73